tests/integration/targets/route53_zone/tasks/main.yml

```diff
@@ -42,6 +45,8 @@
         zone: "{{ resource_prefix }}.check.public"
         comment: original comment
         state: present
+        tags:
+          TestTag: "{{ resource_prefix }}"
```

Comment:
can you also replace `TestTag` with a different Keyname using `purge_tags: yes`?  
Otherwise `purge_tags` is only tested like "remove all keys".

Reply:
Done.